index.html

```diff
@@ -214,10 +214,10 @@ <h3>
               <p>
 This specification defines how to express proofs of integrity, such as digital
 signatures or proofs of existence, for bounded documents, such as verifiable
-credentials. Concrete serializations will be provided for <a
+credentials. Concrete serializations will be provided based on <a
 href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a> and the <a
 href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>.
```

Comment:
```suggestion
credentials. Concrete serializations based on <a
href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a> and the <a
href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a> will
be provided.
```

Reply:
Actually... it would be more appropriate to refer to the TR document, ie, https://www.w3.org/TR/vc-data-model/#json-web-token

(I have not checked whether this is an issue elsewhere)